SPECS/privacyidea-selinux.spec

```diff
@@ -0,0 +1,89 @@
+%global selinuxtype	targeted
+%global moduletype	services
+%global modulenames	privacyidea-selinux	
+
+# Usage: _format var format
+#   Expand 'modulenames' into various formats as needed
+#   Format must contain '$x' somewhere to do anything useful
+%global _format() export %1=""; for x in %{modulenames}; do %1+=%2; %1+=" "; done;
+
+# Relabel files
+%global relabel_files() \ # ADD files in *.fc file
+
+# Version of distribution SELinux policy package
+%if 0%{?centos_ver} == 7
+%global selinux_policyver 3.13.1-266.el7
+%endif
+%if 0%{?centos_ver} == 8
+%global selinux_policyver 3.14.3-20.el8
+%endif
+
+# Version of distribution SELinux policy package 
+#%global selinux_policyver 3.14.3-20.el8
+
+# Package information
+Name:			privacyidea-selinux
+Version:		1.0	
+Release:		1%{?dist}
+License:		GPLv2
+Group:			System Environment/Base
+Summary:		SELinux Policy for privacyidea privacyidea-server 
+BuildArch:		noarch
+URL:			https://privacyidea.org
+Requires(post):		selinux-policy-base >= %{selinux_policyver}, selinux-policy-targeted >= %{selinux_policyver}, policycoreutils, libselinux-utils
+BuildRequires:		selinux-policy selinux-policy-devel
+
+Source:			%{name}-%{version}.tar.gz
+
+%description
+privacyidea-selinux is a additional package 
+for the RPM package privacyidea/privacyidea-server
+with a SELinux policy module, that allows
+httpd service the communicate with the services mysql and ldap
+
+%prep
+%setup -q
+
+%build
+make SHARE="%{_datadir}" TARGETS="%{modulenames}"
+
+%install
+# Install SELinux interfaces
+%_format INTERFACES $x.if
```

Comment:
since we only have one module we can get rid of the `%_format` function and use the explicit file here.

Reply:
If we have more than one module in the future, we should not change this function.